src/translations/nbTranslations.json

```
@@ -1483,7 +1483,7 @@
       "version_accepted_helper_text": "<0>Akseptert versjon</0>: forfatters siste gjennomarbeidede versjon etter at artikkelen har vært gjennom fagfellevurdering, slik den er sendt til forlaget. Denne versjonen har ikke forlagets sidetall og grafiske stil, eller eventuelle redaksjonelle omskrivinger, som den publiserte versjonen har. Denne versjonen bør deles i NVA, helst i pdf-format, med en CC-BY-lisens.",
       "version_accepted_helper_text_metadata_only": "<0>Akseptert versjon</0>: forfatters siste gjennomarbeidede versjon etter at artikkelen har vært gjennom fagfellevurdering, slik den er sendt til forlaget. Denne versjonen har ikke forlagets sidetall og grafiske stil, eller eventuelle redaksjonelle omskrivinger, som den publiserte versjonen har. Denne versjonen bør deles i NVA, helst i pdf-format, med en CC-BY-lisens.",
       "version_embargo_helper_text": "<0>Embargo/utsatt publiseringsdato</0>: De fleste forlag vil kreve at du ikke offentliggjør publisert versjon av artikkelen før det har gått en viss tid. Dato for tillatt offentliggjøring er normalt spesifisert i en forfatteravtale. I noen tilfeller kan også akseptert versjon være underlagt krav om utsatt publiseringsdato.",
-      "version_helper_text": "<0>Last opp akseptert og/eller publisert versjon av artikkelen. Hvis publisert versjon av artikkelen er åpent tilgjengelig, trenger du ikke laste opp akseptert versjon.</0> <1>Publiseringsavtale eller annen formell kommunikasjon (epost) med redaktøren, som avklarer betingelser for utsatt publiseringstidspunkt (embargo) og lisens bør laster opp. <0>Marker slike filer med “Ikke publiser”</0></1>.",
+      "version_helper_text": "<0>Last opp publisert eller akseptert versjon av artikkelen. Om publisert versjon har CC-lisens, så er det ikke nødvendig å laste opp akseptert versjon. Last gjerne også opp publiseringsavtale eller annen formell kommunikasjon (epost) med redaktør, som avklarer betingelser for utsatt publiseringstidspunkt (embargo) eller lisensvilkår. Slike filer markerer du som “Intern” og blir kun synlig for de som har tilgang til å redigere din registrering (dvs deg, medforfattere og impliserte kuratorer/redaktører).</0><0>Din institusjon har gitt deg tilgang til å publisere. Du må sjekke at du har lov til å publisere filene du har lastet opp og at påført lisens er korrekt. Om du trenger hjelp, trykk på “Få hjelp av kurator”-knappen, nederst på siden.<0>",
```

escape fnutter her også?
```suggestion
      "version_helper_text": "<0>Last opp publisert eller akseptert versjon av artikkelen. Om publisert versjon har CC-lisens, så er det ikke nødvendig å laste opp akseptert versjon. Last gjerne også opp publiseringsavtale eller annen formell kommunikasjon (epost) med redaktør, som avklarer betingelser for utsatt publiseringstidspunkt (embargo) eller lisensvilkår. Slike filer markerer du som \"Intern\" og blir kun synlig for de som har tilgang til å redigere din registrering (dvs deg, medforfattere og impliserte kuratorer/redaktører).</0><0>Din institusjon har gitt deg tilgang til å publisere. Du må sjekke at du har lov til å publisere filene du har lastet opp og at påført lisens er korrekt. Om du trenger hjelp, trykk på \"Få hjelp av kurator\"-knappen, nederst på siden.<0>",
```

---

Ah, ja. Skal fikse!